src/components/projectcard.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ProjectCard = ({ image, title, text, color }) => (
+  <div className={`project-card ${color}`}>
+    <div className="text-container">
+      <h2 className="project-title">{title}</h2>
+      <hr></hr>

```

Comment:
use self closing tag ([prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) should reformat it that way automatically )